src/deprecatedCompat/deprecate.ts

```diff
@@ -7,12 +7,6 @@ import {
     version,
 } from "./_namespaces/ts";
 
-export let enableDeprecationWarnings = true;

```

Comment:
All of the changes in this file should be marked as ignored too. All of this code has been used in the past and may be used in the future.